web/gds-admin-ui/src/pages/app/dashboard/TasksChart.js

```diff
@@ -0,0 +1,179 @@
+// @flow
+import React from 'react';
+import { Bar } from 'react-chartjs-2';
+import { Card } from 'react-bootstrap';
+import { useSelector } from 'react-redux';
+import { STACKED_BAR_LABEL } from '../../../constants/dashboard';
+
+
+const TasksChart = (): React$Element<any> => {
+    const { reviews, isLoading } = useSelector(state => ({
+        reviews: state.Reviews.data,
+        isLoading: state.Reviews.loading
+    }))
+
+    const getWeeks = () => {
+        if (reviews && Array.isArray(reviews)) {
+            return reviews.map(review => review.week)
+        }
+
+        return []
+    }
+
+    const getData = (key = '') => {
+        if (reviews && Array.isArray(reviews)) {
+            return reviews.map(review => {
+                return review.registrations[key];
+            })
+        }
+
+        return []
+    }
+
+    const barChartData = {
+        labels: [
+            ...getWeeks()
+        ],
+        datasets: [
```

Comment:
Would you mind creating a story for updating this to access data from the API?

Reply:
Alright